.eslintrc

```diff
@@ -0,0 +1,224 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  // includes the typescript specific rules found here: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules
+  "plugins": [
+    "@typescript-eslint",
+    "import",
+    "no-only-tests",
+    "no-relative-import-paths",
+    "prettier"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "prettier"
+  ],
+  "ignorePatterns": [
+    "package.json",
+    "./frontend/src/third_party/mlmd",
+    "./frontend/src/__tests__/cypress/.nyc_output",
+    "./frontend/src/__tests__/cypress/coverage",
+    "./frontend/package.json",
+    "./frontend/config"
+  ],
+  "globals": {
+    "window": "readonly",
+    "describe": "readonly",
+    "test": "readonly",
+    "expect": "readonly",
+    "it": "readonly",
+    "process": "readonly",
+    "document": "readonly"
+  },
+  "settings": {
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts", ".tsx"]
+    },
+    "import/resolver": {
+      "typescript": {
+        "project": "."
+      }
+    }
+  },
+  "rules": {
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/interface-name-prefix": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-function": "warn",
+    "@typescript-eslint/no-inferrable-types": "error",
+    "@typescript-eslint/no-explicit-any": "off",
+    "import/extensions": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "error",
+    "import/no-unresolved": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        "arrowParens": "always",
+        "singleQuote": true,
+        "trailingComma": "all",
+        "printWidth": 100
+      }
+    ]
+    // "jsx-a11y/no-autofocus": ["error", { "ignoreNonDOM": true }],
+    // "jsx-a11y/anchor-is-valid": [
+    //   "error",
+    //   {
+    //     "components": ["Link"],
+    //     "specialLink": ["to"],
+    //     "aspects": ["noHref", "invalidHref", "preferButton"]
+    //   }
+    // ],
+    // "arrow-body-style": "error",
```

Comment:
Instead of commenting out all these rules, move the common rules that we want to support from the frontend to the root. Then in the backend add the same rule as disabled if it's causing issues an we can address them later.

But for a first step this is ok.